lxd/storage_volumes_state.go

```diff
@@ -104,7 +104,8 @@ func storagePoolVolumeTypeStateGet(d *Daemon, r *http.Request) response.Response
 	}
 
 	// Get the storage project name.
-	projectName, err := project.StorageVolumeProject(s.DB.Cluster, request.ProjectParam(r), volumeType)
+	requestProjectName := request.ProjectParam(r)
+	projectName, err := project.StorageVolumeProject(s.DB.Cluster, requestProjectName, volumeType)
```

Comment:
Sometimes those 2 lines are separated by an empty line. I'm fine with either version but maybe we want to always use the same style.

Reply:
where?

Reply:
in the ones ive changed i cant see it :S

Reply:
https://github.com/canonical/lxd/pull/13014/commits/17c39c6bc262a795ad3567aa3df84f9005ca9edc has the empty line in between.

Reply:
https://github.com/canonical/lxd/pull/13014/commits/1984a8b082a96c24dd97c741cd6bf5fe522e374d and https://github.com/canonical/lxd/pull/13014/commits/ad33d6832d623f4a80f01be64ec6a896223e16b2 do not have that empty line.

Reply:
hrm still cant see what you mean, they all have no line between from what i can see.

Reply:
Sigh, I couldn't read the side-by-side diff properly. The line numbers confirmed the empty lines were just a display artifact, sorry!